manim/utils/testing/_frames_testers.py

```diff
@@ -48,6 +49,22 @@ def check_frame(self, frame_number: int, frame: np.ndarray):
             )
             self._frames_compared += 1
         except AssertionError as e:
+            number_of_matches = np.isclose(
+                frame, self._frames[frame_number], atol=1.01
```

Comment:
Nitpick : At this point, I would feel the need to create a constant for both the absolute tolerance (used twice) and the mismatch tolerance.

Reply:
That makes sense. Do you think just including two constants

```
FRAME_ABSOLUTE_TOLERANCE = 1.01
FRAME_NUMBER_MISMATCH_TOLERANCE = 1e-5
```

at the top of `_frame_testers.py` is sufficient? I don't see these constants being used in any other file in `utils/testing`, so for now this might be okay. They aren't really user-facing, so I wouldn't want to put them in `constants.py`.

Reply:
Done via 346690c0